tests/foreman/destructive/test_puppetplugin.py

```diff
@@ -149,13 +149,21 @@ def test_positive_install_configure(session_puppet_enabled_sat, rhel_contenthost
            installs puppet-agent, configures it, runs it to create the puppet cert,
            signs in on the Satellite side and reruns it.
         2. Assert that Config report was created at the Satellite for the content host.
+        3. Assert that Facts were reported for the content host.
 
     :expectedresults:
         1. Configuration passes without errors.
         2. Config report is created.
+        3. Facts are acquired.
+
+    :BZ: 2126891
     """
     rhel_contenthost.configure_puppet(proxy_hostname=session_puppet_enabled_sat.hostname)
     result = session_puppet_enabled_sat.cli.ConfigReport.list(
-        {'search': f'host={rhel_contenthost.hostname},origin=Puppet'}
+        {'search': f'host={rhel_contenthost.hostname}.infra.sat.rdu2.redhat.com,origin=Puppet'}
```

Comment:
Please check if this hostname is correct, and make sure to fetch this hostname from settings if required

Reply:
Yes, it is how the host is listed in reports. Passed locally.

Reply:
@vsedmik Gotcha, and for fetching this hostname from settings, I meant from dynaconf, we shouldn't put domain names publicly, right?